imports/server/activityTracking.ts

```diff
@@ -0,0 +1,118 @@
+import { Meteor } from 'meteor/meteor';
+import { MongoInternals } from 'meteor/mongo';
+import { Random } from 'meteor/random';
+import { ACTIVITY_GRANULARITY } from '../lib/config/activityTracking';
+import PuzzleActivities from './models/ConsolidatedActivities';
```

Comment:
nit: Maybe rename this import to `ConsolidatedActivities`?

Reply:
Bah yes, that's an oversight - I renamed the collection a few times while figuring out the flow. Will fix